src/ui/page/file/view.jsx

```diff
@@ -22,15 +20,15 @@ import getMediaType from 'util/get-media-type';
 import RecommendedContent from 'component/recommendedContent';
 
 type Props = {
```

Comment:
It looks like this component both receives the entire `Claim` and properties of the claim independently. Should it always be one or the other?

Reply:
I think we should start moving to bring in individual properties that components use: see `makeSelectTitleForUri`, instead of large objects if the component only uses a couple of values.

It's faster for react to update because then it isn't doing tons of un-needed checks to see if an object changed at all.